ui/candidate-portal/src/app/components/form/family-doc-form/family-doc-form.component.ts

```diff
@@ -0,0 +1,248 @@
+  @Input() readOnly = false;
+  @Input() candidate: Candidate;
+  @Output() submitted = new EventEmitter<FamilyDocFormData>();
+
+  form: FormGroup;
+  error: any = null;
+  submitting = false;
+
+  // existing
+  relationships: EnumOption[] = enumOptions(DependantRelations);
+
+  constructor(
+    private fb: FormBuilder,
+    private candidateFormService: CandidateFormService
+  ) {
+  }
+
+  ngOnInit(): void {
+    this.form = this.fb.nonNullable.group({
+      noEligibleFamilyMembers: [false, []],
+      noEligibleNotes: [{value: '', disabled: false}],
+      members: this.fb.nonNullable.array([] as FormGroup[])
+    });
+
+    // "no eligible" toggle
+    this.form.get('noEligibleFamilyMembers')!.valueChanges.subscribe(checked => {
+      const members = this.members;
+      const notesCtrl = this.form.get('noEligibleNotes')!;
+      if (checked) {
+        members.disable({emitEvent: false});
+      } else {
+        notesCtrl.clearValidators();
+        members.enable({emitEvent: false});
+      }
+      notesCtrl.updateValueAndValidity({emitEvent: false});
+    });
+
+    // Load existing
+    this.error = null;
+    this.candidateFormService.getFamilyDocsForm().subscribe({
+      next: data => this.hydrateForm(data),
+      error: () => this.form.reset({noEligibleFamilyMembers: false, noEligibleNotes: ''})
+    });
+  }
+
+  get members(): FormArray<FormGroup> {
+    return this.form.get('members') as FormArray<FormGroup>;
+  }
+
+  private newDocGroup(value?: Partial<FamilyMemberDoc>) {
+    return this.fb.nonNullable.group({
+      docType: [value?.docType ?? TravelDocType.Passport, [Validators.required]],
+      docNumber: [value?.docNumber ?? '', [Validators.required, Validators.maxLength(64)]],
+      issuer: [value?.issuer ?? '', [Validators.required, Validators.maxLength(128)]],
+      issuedOn: [value?.issuedOn ?? '', [Validators.required]],
+      expiresOn: [value?.expiresOn ?? '', [Validators.required]]
+    }, {validators: [this.expiryMinMonthsValidator(9)]});
+  }
+
+  // ✅ UPDATED: Added the new dependant-* fields
+  private newMemberGroup(value?: Partial<RelocatingFamilyMember>) {
+    const group = this.fb.nonNullable.group({
+      relationship: [value?.relationship ?? DependantRelations.Partner, [Validators.required]],
+      dependantRelationOther: [(value as any)?.dependantRelationOther ?? ''],
+
+      firstName: [value?.firstName ?? '', [Validators.required, Validators.maxLength(100)]],
+      lastName: [value?.lastName ?? '', [Validators.required, Validators.maxLength(100)]],
+      dateOfBirth: [value?.dateOfBirth ?? '', [Validators.required]],
+      gender: [value?.gender ?? 'other', [Validators.required]],
+      countryOfBirth: [value?.countryOfBirth ?? '', [Validators.required]],
+      placeOfBirth: [value?.placeOfBirth ?? ''],
+
+      // NEW fields
+      dependantHealthConcerns: [(value as any)?.dependantHealthConcerns ?? null],
```

Comment:
Why prefix these fields with "dependant"? You don't prefix other fields - eg dependantFirstName.

The publishing code that I wrote depends on dependant fields having the same names are the fields for the primary applicant. That way dependant data can be displayed in the same published columns as the corresponding fields for the primary applicant.